src/app/add-item/add-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-add-item',
+  templateUrl: './add-item.component.html',
+  styleUrls: ['./add-item.component.scss'],
+})
+
+export class AddItemComponent implements OnInit {
+  items = [];
+  // itemList -> chenge next whit the list of lists for choice one
+  itemList = 'example';
+  itemID: number = this.items.length + 1;
```

Comment:
unnecessary type definition, already inferred.
Moreover the this definition do not guarantee unique ids. try to generate a unique id or use a library that does it.

Since you always increment this, there is no need to use `items.length`.
This field seems to be private.